README.md

```diff
@@ -99,3 +99,9 @@ cd docs
 yarn
 yarn start
 ```
+
+To serve on a local network (replace `0.0.0.0` with your machine's local IP address):
```

Comment:
```suggestion
You can use the `--host 0.0.0.0` argument to make the dev server available on your local network:
```

Reply:
It took me a while to figure out that you (read: I) should put exactly your local IP, you cannot create a new one. That's why I put the comment.

Reply:
Yes, it took me some searching as well. I'd expect that adding `--host` was enough, but it's not 😄 

But you don't have to specify a local IP; by specifying `0.0.0.0`, the server is bound to the local network. Using `localhost` as the value will only work on your own machine.